src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -119,118 +52,80 @@ public SimulcastController(MediaStreamTrackDesc source, long targetSSRC)
     @Override
     public Bitrates getBitrates()
     {
-        long currentBps = 0;
-        MediaStreamTrackDesc source = weakSource.get();
-        if (source == null)
-        {
-            return Bitrates.EMPTY;
-        }
-
-        RTPEncodingDesc[] sourceEncodings = source.getRTPEncodings();
-        if (ArrayUtils.isNullOrEmpty(sourceEncodings))
-        {
-            return Bitrates.EMPTY;
-        }
-
-        int currentIdx = context.currentIdx;
-        if (currentIdx != -1)
-        {
-            if (sourceEncodings[currentIdx].isActive())
-            {
-                currentBps = sourceEncodings[currentIdx].getLastStableBitrateBps();
-            }
-        }
-
-        long optimalBps = 0;
-        int optimalIdx = this.optimalIdx;
-        if (optimalIdx >= 0)
-        {
-            for (int i = optimalIdx; i > -1; i--)
-            {
-                if (!sourceEncodings[i].isActive())
-                {
-                    continue;
-                }
-
-                long bps = sourceEncodings[i].getLastStableBitrateBps();
-                if (bps > 0)
-                {
-                    optimalBps = bps;
-                    break;
-                }
-            }
-        }
-
-        return new Bitrates(currentBps, optimalBps);
+        return bitstreamController.getBitrates();
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public long getProtectedSSRC()
+    public long getTargetSSRC()
     {
-        return targetSSRC;
+        return bitstreamController.getTargetSSRC();
     }
 
     /**
-     * Defines a packet filter that controls which packets to be written into
-     * some arbitrary target/receiver that owns this {@link SimulcastController}.
-     *
-     * @param buf the <tt>byte</tt> array that holds the packet.
-     * @param off the offset in <tt>buffer</tt> at which the actual data begins.
-     * @param len the number of <tt>byte</tt>s in <tt>buffer</tt> which
-     * constitute the actual data.
-     * @return <tt>true</tt> to allow the specified packet/<tt>buffer</tt> to be
-     * written into the arbitrary target/receiver that owns this
-     * {@link SimulcastController} ; otherwise, <tt>false</tt>
+     * {@inheritDoc}
      */
-    public boolean accept(byte[] buf, int off, int len)
+    @Override
```

Comment:
The docs for SimulcastController#accept are missing from the current tip of the PR